CARDS-1692: AUDIT-C form marked as complete when automatically created

acrowthe opened the pull request:
- Create a blank score answer whenever automatically copying the sex into AUDIT-C so that the form will not get incorrectly marked as complete

![image](https://user-images.githubusercontent.com/20056751/157735181-7718a609-4076-481c-bd25-35d6899e95ed.png)
![image](https://user-images.githubusercontent.com/20056751/157735268-cfcf1995-7794-4e79-908b-50f89bb1073e.png)

acrowthe committed:
CARDS-1692: AUDIT-C form marked as complete when automatically created
- Create a blank score answer whenever automatically copying the sex into
  AUDIT-C so that the form will not get incorrectly marked as compelte

acrowthe committed:
CARDS-1692: AUDIT-C form marked as complete when automatically created
- Fix issue where score answer was not being created in the correct section

sashaandjic reviewed (approved):
tested
Before the change AUDIT-C would be created and marked as completed if gender was specified in the Patient form
![image](https://user-images.githubusercontent.com/23243244/157744618-58e325f1-bb10-4744-b2a9-f6698b80b0d9.png)
Now, AUDIT-C forms get created but it does not get marked as competed:
![image](https://user-images.githubusercontent.com/23243244/157744689-5ebf7f56-9fb3-4bd4-8a13-e1c868df44ac.png)

sdumitriu reviewed (changes requested):
Regression: 
- start in proms mode
- start and configure the mock torch server
- import
- generate a token and log in as the patient
- fill in all the forms -> AUDIT-C is always marked as incomplete, submission is not possible, reviewing is futile.

sashaandjic commented:
Regression confirmed
![image](https://user-images.githubusercontent.com/23243244/157805582-1af5bba0-1b93-472d-9d9e-2bdbc3cb6f6f.png)
![image](https://user-images.githubusercontent.com/23243244/157805594-efbb78e7-9af2-45e9-bff0-cfae0a853c5b.png)

acrowthe committed:
CARDS-1692: AUDIT-C form marked as complete when automatically created
- Fix error where AUDIT-C forms where not getting marked as complete
  when filled out using the PROMs UI

acrowthe commented:
Question being created with no answer has been swapped from audit_score to audit_1, which should resolve the INCOMPLETE errors

sdumitriu commented on proms-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/AUDITC.xml:
```diff
@@ -96,6 +96,11 @@ disorder. AUDIT-C has been validated in mental health and primary care clinics.
 				<value>formatted</value>
 				<type>String</type>
 			</property>
+			<property>
```
@acrowthe shouldn't this be moved to another question?

marta- replied:
It can be removed altogether. `audit_1` which we are creating an answer node for now already has `minAnswers=1` .

sashaandjic reviewed (approved):
tested

acrowthe committed:
CARDS-1692: AUDIT-C form marked as complete when automatically created
- Cleanup audit_score being required

sashaandjic reviewed (approved):
re-tested - all good
